### PR TITLE
Lines within 120 characters for several documents. Ref 73. 

### PR DESCRIPTION
**Pull request by Jigidy**
Ref https://github.com/OpenMS/OpenMS-docs/issues/73.

## Describe the change

Fixed the introduction.md doc to have all lines fit within 120 characters. 
Screenshot 2022-06-18 at 2 42 20 PM
Screenshot 2022-06-18 at 2 42 39 PM

## PR checklist

- [X] I have added description of the change I'm proposing in the OpenMS Documentation.
- [X] I have read and followed [OpenMS Documentation Contributing guidelines](CONTRIBUTING.md).
- [X] I have attached a screenshot of the relevant area after this change.
- [x] `CHANGELOG.md` is updated.
- [x] I have added my name in [CONTRIBUTING.md](CONTRIBUTING.md#openms-documentation-contributors).

### CODE REVIEW

File: docs/introduction.md [hunk 1]
```diff
@@ -96,7 +97,8 @@ Get started with installing OpenMS using the installers available for different
 ```{tab} GNU/Linux
 
 ```bash
-wget https://abibuilder.informatik.uni-tuebingen.de/archive/openms/OpenMSInstaller/release/latest/OpenMS-2.8.0-Debian-Linux-x86_64.deb
+wget https://abibuilder.informatik.uni-tuebingen.de/archive/openms/OpenMSInstaller/release/latest/OpenMS-2.8.0-Debian-
+Linux-x86_64.deb
```
Comment (tapaswenipathak) [inline docs/introduction.md]:
```suggestion
wget https://abibuilder.informatik.uni-tuebingen.de/archive/openms/OpenMSInstaller/release/latest/OpenMS-2.8.0-Debian-Linux-x86_64.deb
```

**Review by tapaswenipathak — CHANGES_REQUESTED** [review]
minor edit plus please update the pr title. let's leave any URLs or group terms.

(always ref the issue.)

**Comment by Jigidy** [comment]
Sounds good. I will leave the links and terms alone.

File: docs/introduction.md [hunk 2]
```diff
@@ -82,12 +82,13 @@ The measurement of one sample is called an MS run and the set of spectra called
...
-based on the {term}`MS(1)` mass-to-charge and the isotope pattern is highly ambiguous. To improve identification, tandem mass
-spectrometry {term}`(MS/MS)` can be applied to assess the analyte substructure. To this end, the precursor ion is isolated and
-kinetically fragmented using an inert gas (e.g., Argon). Fragments produced by {term}`collision-induced dissociation (CID)`
-are stored in an {term}`MS^2` (or {term}`MS/MS`) spectrum and provide information that helps to resolve the ambiguities in identification.
-Alternatively, {term}`MS/MS` spectra can be used for quantification.
+In proteomics and metabolomics, the {term}`MS(1)` intensity is often used for the quantification of an analyte. 
+Identification based on the {term}`MS(1)` mass-to-charge and the isotope pattern is highly ambiguous. To improve 
+identification, tandem mass spectrometry {term}`(MS/MS)` can be applied to assess the analyte substructure. To this end,
+the precursor ion is isolated and kinetically fragmented using an inert gas (e.g., Argon). Fragments produced by {term}
+`collision-induced dissociation (CID)` are stored in an {term}`MS^2` (or {term}`MS/MS`) spectrum and provide information
```
Comment (ghost) [inline docs/introduction.md:89]: why does the user need to know about CID? are there OpenMS algorithms that are based on CID? I think a simplified explanation of what mass spectrometry is, is required.

Comment (tapaswenipathak) [reply docs/introduction.md:89]: i was reading and felt the need. please feel free, to ticket or create a pull request.

**Review by tapaswenipathak — APPROVED** [review]

**Pull request merged**